DMCompiler/DM/Expressions/LValue.cs

```diff
@@ -41,6 +41,17 @@ public Src(Location location, DreamPath? path)
         }
     }
 
+    // opendream_procpath
+    class OpenDreamProcpath : LValue {
```

Comment:
This shouldn't be an LValue since it's not an assignable value. This could probably be a normal `DMExpression` that pushes the proc's path at compile-time. Maybe give it a `TryAsConstant()` too?